js/map_utils.js

```diff
@@ -117,43 +117,42 @@ function updateView(newView, fireAnalytics) {
     updateStateSelector(activeView);
   } 
   
+  // update the geospatial data for the upcoming resolution
+  updateCounties(activeView);
+  updateStates(activeView);
+  
+  // ensure we have the zoom parameters (they're in the state zoom data) and apply the zoom
+  updateStateData(newView, function() {
+    applyZoomAndStyle(newView);
+  });
+  
+  // record the change for analytics. don't need timeout for view change   
+  if(fireAnalytics) {
+    gtag('event', 'update view', {
+      'event_category': 'figure',
+      'event_label': 'newView=' + newView + '; oldView=' +     oldView + '; category=' + activeCategory
+    });
+  }    
+}
+
+function applyZoomAndStyle(newView) {
```

Comment:
i've pulled the code in `applyZoomAndStyle` out from the body of `updateView` so that it can be nested as a callback to updateStateData (lines 125-127 above, https://github.com/USGS-VIZLAB/water-use-15/pull/224/files#diff-2afd72a00f31d3d53312dce9ff03a875R125). This allows us to ensure that the high-resolution state data, which have the high-resolution=correct zoom parameters, are available for use in zooming.